releasenotes/notes/0.24/fix-memory-commutation-checker-dbb441de68706b6f.yaml

```diff
@@ -1,9 +0,0 @@
----
-fixes:
```

Comment:
Are these release notes that are being moved elsewhere or removed entirely? This one e.g. seems like a good one to include somewhere, but I didn't see it elsewhere in the diff.

Reply:
I removed files that were included in earlier bugfix releases. This one was released in 0.23.3: https://github.com/Qiskit/qiskit-terra/blob/0.23.3/releasenotes/notes/fix-memory-commutation-checker-dbb441de68706b6f.yaml